### PR TITLE
13911 create a restful service version for status report

### PR DESCRIPTION
**Pull request by saricicekibrahim**

### CODE REVIEW

File: report/app.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                  // find last completed time
+                  for (var i = jsonRec.eventDetails.length - 1; i >= 0; i--) {
+                    if (jsonRec.eventDetails[i].eventType === finishTxt) {
+                      jsonRec.lastEventTime = jsonRec.eventDetails[jsonRec.eventDetails.length - 1].date;
+                    }
+                    break;
+                  }
+
+                  //chack if job failed
+                  let failedFile = "./logs/" + rec[recLog] + "/" + rec[recEventName] + "/" + logFileName
```
Comment (Zkir) [inline report/app.js:98]: it seems that this is wrong.
it is assumed that app.js is run from /home/user-name/geocint  folder, but it cannot be so.
This folder itself is created by geocint runner an may not exist before the first run.

App should find proper path to logs, when run from geocint-runner folder.

File: report/reportFiles/script.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    window.location.reload(1);
+}, 1000 * 5 * 60);
+
+function doRequest() {
+
+    // Creating Our XMLHttpRequest object 
+    var xhr = new XMLHttpRequest();
+
+    // Making our connection  
+    var url = "http://" + window.location.hostname + '/statusReport';
```
Comment (Zkir) [inline report/reportFiles/script.js:48]: we had to change is line to var 
`url = "http://localhost:5000/statusReport";`

not sure is it important or not.

**Review by Zkir — CHANGES_REQUESTED** [review]

**Comment by Komzpa** [comment]
I am concerned about this development: somehow we ended up with a new backend service skipping general architecture review and having a very misplaced code. 

The svg report generator is profile_make and the HTML report is also expected to be placed there, if developed as static generator.

If we want to do this not in a static fashion but rather as a web app, we need to utilize our frontend/backend framework and rather make it a custom disaster ninja app, with proper data routing. 

This code has to be re-done in either of above ways. Current architecture of the above is not acceptable for long term management.

**Comment by saricicekibrahim** [comment]
no restful service needed

**Pull request closed without merging**